### PR TITLE
Add new success criteria link

### LINKED ISSUE #507: Missing WCAG link
## Describe the bug

**To Reproduce**

1. Go to any page
2. Run Accessibility Insights for Web > Assessment
3. Go to Multimedia > Audio Description > more info and examples
4. missing link for WCAG 1.2.3

## Expected behavior

Add a link in "WCAG Success criteria" for this requirement.
Name:  Understanding Success Criterion 1.2.3: Audio Description or Media Alternative (Prerecorded)
link: https://www.w3.org/WAI/WCAG21/Understanding/audio-description-or-media-alternative-prerecorded

## Screenshots
.
image

## Extension (please complete the following information)

NA

## Are you willing to submit a PR?

If asked, will you be willing to submit a PR to fix this bug? No

## Did you search for similar existing issues?
Yes

### PR DESCRIPTION
**Pull request by smoralesd**
#### Description of changes

Add missing success criteria link on Multimedia -> Audio Description content

01 - new link added

#### Pull request checklist

- [x] Addresses an existing issue: Fixes #507 
- [ ] Added relevant unit test for your changes. (`npm run test`)
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] Ran precheckin (`npm run precheckin`)
- [x] Added screenshots/GIFs for UI changes.

### CODE REVIEW

File: src/content/test/multimedia/audio-description.tsx [hunk 1]
```diff
@@ -53,6 +53,9 @@ export const infoAndExamples = create(({ Markup }) => (
             <Markup.HyperLink href="https://www.w3.org/WAI/WCAG21/Understanding/audio-description-prerecorded">
                 Understanding Success Criterion 1.2.5: Audio Description (Prerecorded)
             </Markup.HyperLink>
+            <Markup.HyperLink href="https://www.w3.org/WAI/WCAG21/Understanding/audio-description-or-media-alternative-prerecorded">
+                Understanding Success Criterion 1.2.3: Audio Description or Media Alternative (Prerecorded)
```
Comment (dbjorge) [inline src/content/test/multimedia/audio-description.tsx]: Is the "1.2.5, then 1.2.3" ordering intentional? I'd have expected it to be listed in order

Comment (smoralesd) [reply src/content/test/multimedia/audio-description.tsx]: That's a good catch. @ferBonnin should I sort this links as per @dbjorge comment?

Comment (ferBonnin) [reply src/content/test/multimedia/audio-description.tsx]: we usually do in order. So 1.2.3 should go first :)

Comment (smoralesd) [reply src/content/test/multimedia/audio-description.tsx]: @dbjorge Updated!

**Pull request merged**